WordPress/gutenberg
# Fix paste handler for MS Word comments/footnotes

## Description
A fix for copy/pasting content from an MS Word document containing comments and/or footnotes. Without this fix, pasting content with comments/footnotes fails silently.

## How has this been tested?
1. Download [clipboard_test_with_footnote.docx.zip](https://github.com/WordPress/gutenberg/files/3658793/clipboard_test_with_footnote.docx.zip), unzip, open in MS Word (tested on macOS version only)
2. Select all content, copy to clipboard
3. Attempt to paste into `/wp-admin/post-new.php` in current version of WordPress (it should fail)
4. Run `npm run build` on this branch
5. In your WordPress install, replace the file `/wp-includes/js/dist/blocks.min.js` with `build/blocks/index.js`
6. Reload `/wp-admin/post-new.php` and try to paste again (it should work)

See also: https://github.com/WordPress/gutenberg/issues/16936

## Types of changes
* Bug fix

## Checklist:
